ui/apps/platform/src/Containers/Clusters/InitBundles/InitBundleWizardStep1.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { ReactElement } from 'react';
+import {
+    Alert,
+    Flex,
+    Form,
+    FormGroup,
+    Radio,
+    Select,
+    SelectOption,
+    TextInput,
+    Title,
+} from '@patternfly/react-core';
+
+import FormLabelGroup from 'Components/PatternFly/FormLabelGroup';
+import useSelectToggle from 'hooks/patternfly/useSelectToggle';
+
+import {
+    InitBundleWizardFormikProps,
+    installationOptions,
+    nameOfStep1,
+    platformOptions,
+} from './InitBundleWizard.utils';
+
+export type InitBundleWizardStep1Props = {
+    formik: InitBundleWizardFormikProps;
+};
+
+function InitBundleWizardStep1({ formik }: InitBundleWizardStep1Props): ReactElement {
+    const { errors, handleBlur, setFieldValue, touched, values } = formik;
+    const installationToggle = useSelectToggle();
+
+    function onChange(value, event) {
+        return setFieldValue(event.target.id, value);
+    }
+
+    return (
+        <Flex direction={{ default: 'column' }}>
+            <Title headingLevel="h2">{nameOfStep1}</Title>
```

Comment:
Are we expecting these title values to change from step to step? If not, I think it would be more straight forward to just put the text in the component itself

Reply:
What do you think of this pattern for consistent title in wizard sidebar and step component?
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ModifyVulnReport/CreateVulnReportPage.tsx#L71-L87

Reply:
Change to more straightforward solution recommended by Linda and seconded by Brad.